jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -587,8 +619,37 @@ public void removeListener(Listener listener)
     public void doStart() throws Exception
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("Scanner start: rprtExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
-                      _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+            LOG.debug("Scanner start: autoStartScanning={}, reportExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
+                isAutoStartScanning(), _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+
+        // Start the scanner and managed beans (eg: the scheduler)
+        super.doStart();
+
+        if (isAutoStartScanning())
+        {
+            startScanning();
+        }
+    }
+
+    /**
+     * Start the scanning process.
+     * <p>
+     *     This will perform the scan of the directories {@link #setScanDirs(List)}
+     *     and schedule future scans, following all of the configuration
+     *     of the scan (eg: {@link #setReportExistingFilesOnStartup(boolean)})
+     * </p>
```

Comment:
```suggestion
     * <p>
     *     This will perform the initial scan of the directories {@link #setScanDirs(List)}
     *     and schedule future scans, following all of the configuration
     *     of the scan (eg: {@link #setReportExistingFilesOnStartup(boolean)})
     * </p>
```